fundamentals/installation.md

```diff
@@ -11,9 +11,23 @@ and be able to make the napari viewer appear.
 
 ## installation
 
+### which distribution to install
+If you want to contribute back to napari codebase, you need to install from source code: see the [from source](#from-source) section.
```

Comment:
Can we present the three installation options in the order we want new users to try them in? (And then have those section headings appear in that order, too)

I'd suggest:
1. from pip or conda-forge
2. bundled app
3. from source

Reply:
That order makes sense to me

Reply:
the rationale for the current ordering is that we want to encourage people without python experience to install from bundled app first, instead of going through the pip approach which requires programming knowledge.

Reply:
i think it's still best to install from pip if it works - it enables jupyter interaction, script running, and is easier to upgrade, so I think I'd prefer to see that listed first. I see the bundled app as a bit of a fall back.